<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>442</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>442</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/login-Background.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
color:white;
border-image:url();
border:1px solid white;
border-radius:20px
}

QPushButton:hover{
color:#B1ACB0;
border:1px solid #B1ACB0;
border-radius:20px
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMinimize">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image:url(:/images/loginBtn-Minimize-Normal.png);
}
QPushButton:hover{
border-image:url(:/images/loginBtn-Minimize-Hover.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>415</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image:url(:/images/loginBtn-Close-Normal.png);
}
QPushButton:hover{
border-image:url(:/images/loginBtn-Close-Hover.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblHint">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url();
background-color:transparent;
border:0px;
color:red
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>190</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url();
background-color:transparent;
border:0px;
color:white
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="account">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url();
background-color:transparent;
border:0px;
color:white
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="sbFactoryMode" native="true">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>77</y>
        <width>94</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>94</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url()</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>80</y>
        <width>72</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url();
color:white</string>
      </property>
      <property name="text">
       <string>工厂模式</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>account</tabstop>
  <tabstop>password</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnMinimize</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
